libgetgood/updater.go

```diff
@@ -48,7 +48,9 @@ func (updater *Updater) work() {
 	for running {
 		select {
 		case r := <-updater.requestChan:
+			log.Printf("Adding URLs to databse\n")
 			err := updater.addURLs(r.Url)
+			log.Printf("URLs Added to database\n")
```

Comment:
I think we can drop these two log statements as they were mainly for debugging the issue.

If we want to include them later I think we should put them in the addURLs function as it's called elsewhere as well.